build/3-apillon-api.md

```diff
@@ -1158,3 +1158,714 @@ curl --location --request GET "https://api.apillon.io/hosting/websites/:websiteU
+| status           | `number`  | Collection status.                                                                                |
+| projectUuid      | `string`  | Apillon project id.                                                                               |
+| symbol           | `string`  | NFT collection symbol (usually 3-4 characters long).                                              |
+| name             | `string`  | NFT collection name.                                                                              |
+| description      | `string`  | NFT collection description.                                                                       |
+| maxSupply        | `number`  | Maximal number of NFTs ever in existence.                                                         |
+| mintPrice        | `number`  | Price of NFT at mint stage.                                                                       |
+| bucketUuid       | `string`  | UUID of the bucket where metadata is stored.                                                      |
+| baseUri          | `string`  | Base URI for collection metadata (token id and file extension is appended to it).                 |
+| baseExtension    | `string`  | File extension that is auto appended after token id to form a full URL.                           |
+| isDrop           | `boolean` | Determines if collection is mintable by public.                                                   |
+| dropStart        | `number`  | UNIX timestamp which determines public mint opening date and time.                                |
+| isSoulbound      | `boolean` | Soul bound tokens are NFTs that are bounded to wallet and not transferable.                       |
+| isRevokable      | `boolean` | For revocable collection owner can destroy NFTs at any time.                                      |
+| reserve          | `number`  | Amount of NFTs reserved by owner.                                                                 |
+| royaltiesFees    | `number`  | Percentage (between 0 and 100) of each NFT sale sent to wallet specified under royalties address. |
+| royaltiesAddress | `string`  | Address where royalties are sent to.                                                              |
+| chain            | `number`  | Blockchain id on which you want to release your collection.                                       |
+| contractAddress  | `string`  | Smart address of contract for deployed collection.                                                |
+| transactionHash  | `string`  | Deployment transaction hash/id.                                                                   |
+| deployerAddress  | `string`  | Wallet address of deployer.                                                                       |
+| minted           | `number`  | Number of NFTs minted for this collection.                                                        |
```

Comment:
We could possibly remove this, since this means checking on chain everytime when user fetches data which could delay requests. This is a thing that could be supported via subsquid indexer that would be deployed per collection as a paid service for the user.